back/app/controllers/web_api/v1/ideas_controller.rb

```diff
@@ -48,7 +48,7 @@ def index
   def index_mini
     ideas = IdeasFinder.new(
       params,
-      scope: policy_scope(Idea).where(publication_status: 'published'),
+      scope: policy_scope(Idea).submitted_or_published,
```

Comment:
Making this consistent with the normal `index` action. The frontend can still filter on publication_status.